pkg/snapshot/velero.go

```diff
@@ -349,13 +350,13 @@ DeploymentFound:
 					status = "Ready"
 				}
 			}
-			veleroStatus.ResticVersion = matches[4]
-			veleroStatus.ResticStatus = status
+			veleroStatus.NodeAgentVersion = matches[4]
+			veleroStatus.NodeAgentStatus = status
 
-			goto ResticFound
+			goto NodeAgentFound
 		}
 	}
-ResticFound:
+NodeAgentFound:
 
 	return &veleroStatus, nil
```

Comment:
Not really part of this PR, but can this (and the one above) just be a `break` since it's not a nested loop?

Reply:
yeah it can, i just didn't wanna touch this logic :)